### DISCUSSION TITLE
Issue with email alerts

### DISCUSSION THREAD

**Discussion by killmasta93** [utmstack/UTMStack#467] [Q&A] [answered]
**Describe the bug**
Not getting email alerts, Already configure the smtp and tested as i got the email 

![image](https://github.com/utmstack/UTMStack/assets/13953629/141a935d-a328-43f9-82b3-0c61e52fa29e)

**Comment by c3s4rfred** [comment]
Hi, @killmasta93. The configuration testing only tests your server email configuration, the alert's recipients are configured in the Alerts settings section, see the image below: 
![image](https://github.com/utmstack/UTMStack/assets/17603499/b7005f98-8b10-4c05-9005-d93ae13b384c)

**Comment by osmontero** [comment]
Moved this to Discussions because cannot be reproduced and it looks like a missing configuration.

**Comment by killmasta93** [comment]
hi @c3s4rfred Thanks for the reply correct i have it configured but no alert was generated when i got a high 

![image](https://github.com/utmstack/UTMStack/assets/13953629/b8b5e19d-6285-4401-9498-4f258890171a)

![image](https://github.com/utmstack/UTMStack/assets/13953629/2676a725-fb87-4675-96c8-0ad2354dc16e)

![image](https://github.com/utmstack/UTMStack/assets/13953629/cda0a801-4adf-4622-bfe8-7ac07deb20b0)

**Reply by c3s4rfred** [reply]
Hi @killmasta93, if the alerts you show in the image after you configure the alerts and incident recipient's email address were the first registered in your UTMStack instance, wait a little bit for new alerts, the email should arrive if there are no network issues or rules blocking this kind of email. Let us know the results.

**Reply by leonardomoralopez89** [reply]
Hi @killmasta93, have you received any additional alerts after the ones you presented to us earlier?

**Reply by killmasta93** [reply]
hi @leonardomoralopez89 @c3s4rfred yeah i got alerts on the dashboard but its not sending any emails

![image](https://github.com/utmstack/UTMStack/assets/13953629/052963ca-0f01-4d5a-a6de-06e4876fa864)

**Reply by leonardomoralopez89** [reply]
Hi @killmasta93. Are those alerts you sent us all the ones you have received, or are there other alerts with other statuses?
Did you receive any example alerts when you installed UTMStack?

**Reply by killmasta93** [reply]
hi @leonardomoralopez89 correct those alerts are new i assume new alerts with high i would get an email right?

**Reply by c3s4rfred** [reply]
Hi, @killmasta93, try to generate new alerts different than the ones that refer to datasources, ex: "The generic datasource installed in .... is taking longer than usual to send logs" and let us know if you receive the Alert's email.

**Reply by killmasta93** [reply]
hi @c3s4rfred thanks for the reply, i was checking the logs and found 
![image](https://github.com/utmstack/UTMStack/assets/13953629/0e1a5eab-414e-43f2-876d-cb808edccbb0)

but still no email alert

**Reply by c3s4rfred** [reply]
Hi @killmasta93, you have to generate new alerts different from than ones, as mentioned in my message before. That kind of alerts is actually omitted from email alerts. In the future will be added as well.

**Reply by killmasta93** [reply]
hi @c3s4rfred quick question how would i generate a new alert? would i do a nmap? to the VM to trigger?

**Reply by killmasta93** [reply]
so i triggered this alert and no email alert
![image](https://github.com/utmstack/UTMStack/assets/13953629/882f0cf3-2e32-4b13-ac73-600b8c73e8df)

**Reply by c3s4rfred** [reply, marked as answer]
Hi, @killmasta93 after deep testing, we discovered a bug related to one of the underlying technologies that causes the bug, which will be solved on the next release, thanks for the feedback.

**Reply by killmasta93** [reply]
thank you
